projects/1.20-fabric/assets/0-modrinth-mod/ubesdelight/lang/zh_cn.json

```
@@ -0,0 +1,89 @@
+  "item.ubesdelight.arroz_caldo": "鸡肉米粥",
+  "item.ubesdelight.bangsilog": "蒜炒饭配煎虱目鱼与煎蛋",
+  "item.ubesdelight.bulalo": "菲式牛骨汤",
+  "item.ubesdelight.chicken_inasal": "菲律宾烤鸡",
+  "item.ubesdelight.chicken_inasal_rice": "菲式烤鸡饭",
+  "item.ubesdelight.condensed_milk_bottle": "炼乳瓶",
+  "item.ubesdelight.cookie_ginger": "生姜饼干",
+  "item.ubesdelight.cookie_ube": "香芋饼干",
+  "item.ubesdelight.fish_sauce_bottle": "鱼露",
+  "item.ubesdelight.garlic": "大蒜",
+  "item.ubesdelight.garlic_chop": "蒜末",
+  "item.ubesdelight.ginger": "生姜",
+  "item.ubesdelight.ginger_chop": "姜末",
+  "item.ubesdelight.halo_halo": "哈喽哈喽刨冰",
+  "item.ubesdelight.kinilaw": "菲律宾酸橘汁腌鱼",
+  "item.ubesdelight.leche_flan": "菲式焦糖布丁切片",
+  "item.ubesdelight.lemongrass": "柠檬香草",
+  "item.ubesdelight.lemongrass_seeds": "柠檬香草种子",
+  "item.ubesdelight.lumpia": "菲律宾润饼",
+  "item.ubesdelight.lumpia_wrapper": "菲律宾润饼皮",
+  "item.ubesdelight.mechado": "菲式茄汁牛肉",
+  "item.ubesdelight.milk_powder": "奶粉",
+  "item.ubesdelight.milk_tea_ube": "香芋奶茶",
+  "item.ubesdelight.poisonous_ube": "毒香芋",
+  "item.ubesdelight.polvorone": "西班牙酥饼",
+  "item.ubesdelight.polvorone_cc": "曲奇奶油味西班牙酥饼",
+  "item.ubesdelight.polvorone_pinipig": "绿糯米味西班牙酥饼",
+  "item.ubesdelight.polvorone_ube": "香芋味西班牙酥饼",
+  "item.ubesdelight.raw_polvorone": "生西班牙酥饼",
+  "item.ubesdelight.raw_polvorone_cc": "生曲奇奶油味西班牙酥饼",
+  "item.ubesdelight.raw_polvorone_pinipig": "生绿糯米味西班牙酥饼",
+  "item.ubesdelight.raw_polvorone_ube": "生香芋味西班牙酥饼",
+  "item.ubesdelight.sinangag": "菲律宾蒜炒饭",
+  "item.ubesdelight.sisig": "肉肉杂碎",
+  "item.ubesdelight.sugar_brown": "红糖",
+  "item.ubesdelight.tocino": "菲式甜烧肉",
+  "item.ubesdelight.tosilog": "蒜炒饭配甜烧肉和煎蛋",
+  "item.ubesdelight.ube": "香芋",
+  "item.ubesdelight.ube_cake_slice": "香芋蛋糕切片",
+  "itemGroup.ubesdelight.group": "香芋乐事",
+  "ubesdelight.container.halo_halo_feast": "需要玻璃瓶。",
+  "ubesdelight.modmenu.category.wildgarlicgen": "野生大蒜生成",
+  "ubesdelight.modmenu.category.wildgingergen": "野生生姜生成",
+  "ubesdelight.modmenu.category.wildlemongrassgen": "野生柠檬香草生成",
+  "ubesdelight.modmenu.category.wildubegen": "野生香芋生成",
+  "ubesdelight.modmenu.option.enableGarlicCompat": "其它模组的大蒜兼容性",
+  "ubesdelight.modmenu.option.enableGingerCompat": "其它模组的生姜兼容性",
+  "ubesdelight.modmenu.option.enableudcropcrates": "香芋乐事箱装作物合成",
+  "ubesdelight.modmenu.option.farmersbuyudcrops": "农民交易香芋乐事作物",
+  "ubesdelight.modmenu.option.foodEffectTooltip": "食物显示提供的效果",
+  "ubesdelight.modmenu.option.generateUDChestLoot": "产生香芋乐事战利品表",
+  "ubesdelight.modmenu.option.wanderingtradersellsuditems": "流浪商人售卖香芋乐事作物种子",
+  "ubesdelight.modmenu.option.wildgen": "生成",
+  "ubesdelight.modmenu.title": "香芋乐事 - 配置界面",
+  "ubesdelight.modmenu.title.clientsettings": "客户端设置",
+  "ubesdelight.modmenu.title.gamesettings": "游戏设置",
+  "ubesdelight.modmenu.title.integration": "集成设置",
+  "ubesdelight.modmenu.title.worldgeneration": "世界生成设置",
+  "ubesdelight.modmenu.tooltip.enableGarlicCompat": "这项设置决定所有在 c:crops/garlic 标签里的作物是否都能被切成蒜末。",
```

```suggestion
  "ubesdelight.modmenu.tooltip.enableGarlicCompat": "这项设置决定所有在c:crops/garlic标签里的作物是否都能被切成蒜末。",
```
不需要英文两侧加空格
此问题请全文通查

---

OK